rocAL_pybind/examples/rocAL_api_audio_unittest.py

```diff
@@ -0,0 +1,84 @@
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import random
+import numpy as np
+from amd.rocal.plugin.pytorch import ROCALAudioIterator
+import torch
+# torch.set_printoptions(threshold=10_000)
+np.set_printoptions(threshold=1000, edgeitems=10000)
+from amd.rocal.pipeline import Pipeline
+import amd.rocal.fn as fn
+import amd.rocal.types as types
+import math
+# import rocal_pybind.tensor
+import sys
+import cv2
+import matplotlib.pyplot as plt
+import os
+def draw_patches(img, idx, device):
```

Comment:
I feel for audio it might not be better to plot the images and shown to user
since it is difficult to interpret, what it is. Would suggest adding QA tests in python to verify the correctness
Otherwise, we can dump the outputs to .csv files if needed, where the numerical data can be shown

Reply:
@shobana-mcw  Please add your thoughts on how we can add QA tests for Audio

Reply:
We need not display it as image since its audio. It does not make sense to have image comparison. 
@sampath1117 How is RPP handling the unit testing for Audio?